main.go

```diff
@@ -78,6 +78,11 @@ func main() {
 			Usage:  "git clone path",
 			EnvVar: "PLUGIN_WORKSPACE,DRONE_WORKSPACE",
 		},
+		cli.BoolTFlag{
```

Comment:
This is bad, this is a breaking change as BoolTFlag is true by default, it had been false before.

Reply:
Shit missed that. Can you just modify the file and correct it in the main branch?

Not in front of a computer